runelite-client/src/main/java/net/runelite/client/plugins/PluginManager.java

```diff
@@ -372,6 +422,15 @@ private Plugin instantiate(Class<Plugin> clazz) throws PluginInstantiationExcept
 			{
 				binder.bind(clazz).toInstance(plugin);
 				binder.install(plugin);
+				for (Plugin p : deps)
+				{
+					Module p2 = (Binder binder2) ->
+					{
+						binder2.bind((Class<Plugin>) p.getClass()).toInstance(p);
```

Comment:
Is it necessary to bind plugin dependency itself here? As it should expose itself as well to the binder (I think) and so I would imagine just calling binder.install(p) would work.

Reply:
It is required since the plugins do not bind themselves.